### ISSUE TITLE
decrypt

### ISSUE THREAD

**Issue by Rapid0nion** [0xbitx/dedsec-wifiphish#1]
I would like to talk to you about the open version.

**Comment by 0xbitx** [comment]
open version?

**Comment by Rapid0nion** [comment]
Yes. I mean decrypted version with open code. Could U text me via get.commerce@proton.me

**Comment by 0xbitx** [comment]
just dm me here https://discord.com/invite/EgkWPws6vV
